src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -23,243 +23,475 @@ internal class BuiltinFunctionsCore
-        public static readonly TexlFunction Tan = _library.Add(new TanFunction());
-        public static readonly TexlFunction TanT = _library.Add(new TanTableFunction());
-        public static readonly TexlFunction Text = _library.Add(new TextFunction());
-        public static readonly TexlFunction Text_UO = _library.Add(new TextFunction_UO());
-        public static readonly TexlFunction Time = _library.Add(new TimeFunction());
-        public static readonly TexlFunction TimeValue = _library.Add(new TimeValueFunction());
-        public static readonly TexlFunction TimeValue_UO = _library.Add(new TimeValueFunction_UO());
-        public static readonly TexlFunction TimeZoneOffset = _library.Add(new TimeZoneOffsetFunction());
-        public static readonly TexlFunction Today = _library.Add(new TodayFunction());
-        public static readonly TexlFunction Trace = _library.Add(new TraceFunction());
-        public static readonly TexlFunction Trim = _library.Add(new TrimFunction());
-        public static readonly TexlFunction TrimEnds = _library.Add(new TrimEndsFunction());
-        public static readonly TexlFunction TrimEndsT = _library.Add(new TrimEndsTFunction());
-        public static readonly TexlFunction TrimT = _library.Add(new TrimTFunction());
-        public static readonly TexlFunction Trunc = _library.Add(new TruncFunction());
-        public static readonly TexlFunction TruncT = _library.Add(new TruncTableFunction());
-        public static readonly TexlFunction UniChar = _library.Add(new UniCharFunction());
-        public static readonly TexlFunction UniCharT = _library.Add(new UniCharTFunction());
-        public static readonly TexlFunction Upper = _library.Add(new LowerUpperFunction(isLower: false));
-        public static readonly TexlFunction UpperT = _library.Add(new LowerUpperTFunction(isLower: false));
-        public static readonly TexlFunction Value = _library.Add(new ValueFunction());
-        public static readonly TexlFunction Value_UO = _library.Add(new ValueFunction_UO());
-        public static readonly TexlFunction VarP = _library.Add(new VarPFunction());
-        public static readonly TexlFunction VarPT = _library.Add(new VarPTableFunction());
-        public static readonly TexlFunction Weekday = _library.Add(new WeekdayFunction());
-        public static readonly TexlFunction WeekdaysLong = _library.Add(new WeekdaysLongFunction());
-        public static readonly TexlFunction WeekdaysShort = _library.Add(new WeekdaysShortFunction());
-        public static readonly TexlFunction WeekNum = _library.Add(new WeekNumFunction());
-        public static readonly TexlFunction With = _library.Add(new WithFunction());
-        public static readonly TexlFunction Year = _library.Add(new YearFunction());
+        public static readonly TexlFunction Abs = __library[0];
+        public static readonly TexlFunction AbsT = __library[1];

```

Comment:
>__library[1] [](http://example.com/codeflow?start=51&length=12)

referencing by index seems extremely brittle. 
Especially if we expect to keep this alpha sorted.

Reply:
if you have an idea, it's always welcome
I'll use an enum to make things a little less brittle but that's unavoidable and a small cost compared to what this PR is adressing (random failures)